config.js

```diff
@@ -113,7 +113,7 @@ CKEDITOR.editorConfig = function ( config ) {
 			attributes: 'align,longdesc,frameborder,height,name,scrolling,src,title,width'
 		},
 		img: {
-			attributes: 'src,alt,title,width,height'
+			attributes: 'src,alt,width,height'
```

Comment:
All changes just roll back of https://github.com/PolicyStat/ckeditor-dev/pull/25